go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
 	github.com/andybalholm/cascadia v1.1.0 // indirect
 	github.com/bmatcuk/doublestar v1.1.5
-	github.com/coveooss/multilogger v0.1.1
+	github.com/coveooss/multilogger v0.2.0-beta.1
```

Comment:
Let's wait for a version not on a branch

Reply:
Just waiting for the official v0.2.0.